.travis.yml

```diff
@@ -1,24 +1,79 @@
-dist: xenial
 language: python
-python:
-- 3.7
-- 3.6
-- 3.5
-- 2.7
-install: pip install -U tox-travis coverage coveralls
+# install python on windows
+
+env:
+  global:
+    # Note: TWINE_PASSWORD is set in Travis settings
+    - TWINE_USERNAME=stevenjeanette
+    # skip py27 (EOL) and linux-32bit (Test get_h->test_nu0 hangs)
+    - CIBW_SKIP="cp27-* *-manylinux_i686"
+    # first versions to support py3.8
+    - CIBW_BEFORE_BUILD="pip install numpy==1.17.3 scipy==1.3.2 mpmath==1.1.0 setuptools"
+    # testing with cibuildwheel
+    - CIBW_TEST_REQUIRES=pytest
+    - CIBW_TEST_COMMAND="python -m pytest -v {project}/tests"
```

Comment:
I *think* this can just be `"pytest -v"`

Reply:
Since the command has to be working on all three OS (Linux, Win, MacOS) I think the ``python`` is needed here.

Reply:
Ah ok. Cool.